include/irr/video/IGPUBufferView.h

```diff
@@ -3,19 +3,29 @@
 
 #include "irr/asset/IBufferView.h"
 #include "IGPUBuffer.h"
+#include "IRenderableVirtualTexture.h"
 #include <utility>
 
 namespace irr {
 namespace video
 {
 
-class IGPUBufferView : public asset::IBufferView<IGPUBuffer>, public core::IReferenceCounted
+class IGPUBufferView : public asset::IBufferView<IGPUBuffer>, public IRenderableVirtualTexture
```

Comment:
no a buffer view is not a texture!

Reply:
(only in OpenGL and under the hood)

Reply:
yea, it's temporary solution. But now I think that I can make only CopenglBufferView inherit from irenderablevirtualtexture. It has to be be like this because of SAuxContext::setActiveTexture()

Reply:
nuke `SAuxContext::setActiveTexture()`

Reply:
why actually?

Reply:
because you'll be setting the entire descriptor set at once, and the descriptors bound in previous bindings determine the GL-bindings of the textures you'll be setting as the result

Reply:
@Crisspl ?

Reply:
resolved